asilvanovich/06.Ansible.Cruise/info.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts : webservers
+  vars:
+    var_01: "Hi"
```

Comment:
If you are not going to use this vars, please remove them

Reply:
sure